BUILD

```diff
@@ -0,0 +1,18 @@
+load("@rules_cc//cc:defs.bzl", "objc_library")
+
+objc_library(
+    name = "Defines",
+    hdrs = ["GTMDefines.h"],
+    visibility = ["//visibility:public"],
+)
+
+objc_library(
```

Comment:
This doesn't seem useful, since the headers are in `srcs` they aren't exposed to deps.  Since there are some backing impls, what's the point of this?  Can we just drop it?

Reply:
Sounds good, I was just being conservative in this PR and matching what we had in the Podspec